### DISCUSSION TITLE
additive blending?

### DISCUSSION THREAD

**Discussion by alex-michaluk** [creativelifeform/three-nebula#227] [Q&A]
Are you able to specify blending mode of particles from the json file?

**Comment by rohan-deshpande** [comment]
It should work, I haven't tried it myself but I think there is a way to define it in `materialProperties` namespace for the texture in the JSON data.

**Reply by alex-michaluk** [reply]
yeah doesn't seem to work - tried this - i just tried subtractive blending (3) to make it obvious:
```
          "type": "BodySprite",
          "properties": {
            "blending": 3,
            "texture": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAgAAAAIACAMAAADDpiTIAAAABGdBTUEAALGPC/
```
            
doesn't show up as an option in the docs either:
https://three-nebula-docs.netlify.app/class/src/initializer/texture.js~texture#instance-member-materialProperties
